src/pages/Schedulers/Tree.tsx

```diff
@@ -0,0 +1,470 @@
+import {
+	ApartmentOutlined,
+	BarChartOutlined,
+	HolderOutlined,
+	PlusCircleOutlined,
+	ShoppingCartOutlined,
+	StarOutlined,
+	SyncOutlined,
+	UsergroupAddOutlined,
+} from '@ant-design/icons';
+import { PopconfirmProps, TreeDataNode } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import Components from '../../components';
+
+type SchedulerTreeProps = {
+	confirmHandler: (identifier: string, method: string) => void;
+	showModal: () => void;
+};
+const SchedulersTree: React.FC<SchedulerTreeProps> = ({ confirmHandler, showModal }) => {
+	const { t } = useTranslation();
+	const confirmPropsValue: PopconfirmProps = {
+		title: t('schedulers.popconfirmTitle'),
+		description: t('schedulers.popconfirmDescription'),
+		okText: t('schedulers.popconfirmOkText'),
+		cancelText: t('schedulers.popconfirmCancelText'),
+	};
+	const LeftSideTreeData: TreeDataNode[] = [
+		{
+			title: t('schedulers.products'),
+			key: '1',
+			icon: <StarOutlined />,
+			children: [
+				{
+					title: (
+						<Components.Popconfirm
```

Comment:
Here all items are sharing the same properties we can create a list in util/constant file and read this and pass all common properties